src/sql/src/session/vars.rs

```diff
@@ -520,6 +520,24 @@ const ENABLE_MULTI_WORKER_STORAGE_PERSIST_SINK: ServerVar<bool> = ServerVar {
     safe: true,
 };
 
+/// The default for the `DISK` option in `UPSERT` sources.
+const UPSERT_SOURCE_DISK_DEFAULT: ServerVar<bool> = ServerVar {
+    name: UncasedStr::new("upsert_source_disk_default"),
+    value: &false,
+    description: "The default for the `DISK` option in `UPSERT` sources.",
+    internal: true,
+    safe: true,
+};
+
+/// Whether or not the `DISK` option in available `UPSERT` sources.
+const UPSERT_SOURCE_DISK_AVAILABLE: ServerVar<bool> = ServerVar {
```

Comment:
This should be named `enable_upsert_source_disk` to match the other feature flag naming conventions.